### DISCUSSION TITLE
Welcome to RWKV-LM Discussions!

### DISCUSSION THREAD

**Discussion by BlinkDL** [BlinkDL/RWKV-LM#174] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pfeatherstone** [comment]
How do you reckon this model compares to [RMT](https://arxiv.org/pdf/2304.11062.pdf) ?

**Reply by PicoCreator** [reply]
Unfortunately, without a model we can 1:1 compare on similar training dataset (ie. enwiki / the pile). There is no way of knowing as of now.

PS: its generally better to ask such questions on our discord - https://discord.gg/bDSBUMeFpc

**Reply by pfeatherstone** [reply]
Ah ok. There is an [implementation](https://github.com/lucidrains/recurrent-memory-transformer-pytorch)

**Reply by wisdoms6969** [reply]
D
